recipes/jupyterlab-flake8/LICENSE

```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
```

Comment:
The license file already appears to be included in the source distribution, so no need to include it again here.